### DISCUSSION TITLE
Inital Observations on the SharpRISCV Assembler

### DISCUSSION THREAD

**Discussion by rtbruce** [rizwan3d/SharpRISCV#10] [General]
1. Support for hexadecimal constants does not seem to be there.
2. Support for pseudo instructions like "li" (Load Immediate), and "mv" (Move register to register) also seems to be missing.  

This is actually not too bad at the moment since knowing those translations is useful to me at this stage.

**Comment by rizwan3d** [comment]
I have started this fun project, and now I am iterating on it using best practices. Check [V2](https://github.com/rizwan3d/SharpRISCV/tree/V2) for updates.

The current version has lots of issues and cannot handle many invalid/valid insertions, including Pseudo instructions.

If you are willing to contribute to the project, you are more than welcome in any capacity—coding, testing, etc.

**Reply by rtbruce** [reply]
That is fine, but I can't do anything with C#, so I'll wait until the next release.

**Reply by rizwan3d** [reply]
Feel free to enjoy the project, explore, and experiment with features as you please.
